theme/templates/resource-landing-page/coverage.html

```diff
@@ -251,13 +251,15 @@ <h4>Spatial:</h4>
                                 </div>
 
                                 {# we will show/hide this button from JS code when ever coverage element gets updated for aggregation #}
-                                <button id="btn-update-resource-spatial-coverage"
-                                        type="button" class="btn btn-info space-top"
-                                        {% if not cm.resource_type == "CompositeResource" or not cm.has_logical_spatial_coverage %}
-                                        style="display: none"
-                                        {% endif %}>
-                                    <i class="fa fa-calculator" aria-hidden="true"></i> Set spatial coverage from content files
-                                </button>
+                                {% if cm.resource_type == "CompositeResource" %}
+                                    <button id="btn-update-resource-spatial-coverage"
```

Comment:
I did notice that the buttons to "Set [spatial/temporal] coverage from content files APPEAR when one starts typing in the related input fields--_regardless of whether the input is "logical" or not._ It seems odd to offer to set coverage while the user it actively typing in a coverage manually. But this is not an issue particular to this PR

Reply:
There are other issues with that button - it should not be displayed when there is no aggregation with spatial coverage. I will create a sperate issue related to this button.

Reply:
sounds great, thanks!